roles/teiserver/templates/teiserver.Dockerfile.j2

```diff
@@ -20,8 +20,8 @@
 #       teiserver
 #
 
-ARG ELIXIR_VERSION=1.14.5
-ARG OTP_VERSION=26.2.3
+ARG ELIXIR_VERSION=1.16.3
+ARG OTP_VERSION=26.2.5.2
 ARG DEBIAN_VERSION=bookworm-20240130
 ARG BUILDER_IMAGE="docker.io/hexpm/elixir:${ELIXIR_VERSION}-erlang-${OTP_VERSION}-debian-${DEBIAN_VERSION}"
```

Comment:
There don't exist any arbitrary combinations of this image, you have to search docker image and pick one that exists, in this case, there is [`hexpm/elixir:1.16.3-erlang-26.2.5.2-debian-bookworm-20240812`](https://hub.docker.com/layers/hexpm/elixir/1.16.3-erlang-26.2.5.2-debian-bookworm-20240812/images/sha256-53f3aa20c297ae6d7a4eb2d85af9f7c2f17902f2775ad5a0712f713dc2bf822f?context=explore) so you also need to adjust `DEBIAN_VERSION` accordingly.